indico/modules/events/editing/controllers/backend/editable_list.py

```diff
@@ -0,0 +1,28 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from sqlalchemy.orm import joinedload
+
+from indico.modules.events.contributions.models.contributions import Contribution
+from indico.modules.events.editing.controllers.base import RHEditableTypeManagementBase
+from indico.modules.events.editing.schemas import EditingEditableListSchema
+
+
+class RHEditableList(RHEditableTypeManagementBase):
+    """Return the list of editables of the event for a given type"""
+    def _process_args(self):
+        RHEditableTypeManagementBase._process_args(self)
+        self.contributions = (Contribution.query
+                              .with_parent(self.event)
+                              .options(joinedload('editables'))
+                              .all())
```

Comment:
We should probably apply a default order here to have something consistent even if the user does not apply any order manually e.g. `.order_by(Contribution.friendly_id)`